### PR TITLE
Fix description of collectwavecoeffs

### PR DESCRIPTION
**Pull request by vanderhe**
At the moment the description listed in the help page of the `collectwavecoeffs` script is wrong, i.e. referring to the `skgen` description.

### CODE REVIEW

File: sktools/src/sktools/scripts/collectwavecoeffs.py [hunk 1]
```diff
@@ -150,6 +151,13 @@ def setup_parser_main(parser):
         help='Logging level (default: info)')
 
 
+def get_parser_and_subparser_container():
+    parser = argparse.ArgumentParser(description=USAGE)
```
Comment (bhourahine) [inline sktools/src/sktools/scripts/collectwavecoeffs.py:155]: Missing a docstring

**Review by bhourahine — APPROVED** [review]
Seems fine, but I wasn't able to work out why a local copy of get_parser_and_subparser_container is required (other than possibly different comments/strings)..

**Comment by vanderhe** [comment]
> Seems fine, but I wasn't able to work out why a local copy of get_parser_and_subparser_container is required (other than possibly different comments/strings)..

It seemed to be the established strategy that at least some scripts provide their own definition. Doesn't mean it's a good design per se...

File: sktools/src/sktools/scripts/collectwavecoeffs.py (hunk 1 above)
Comment (vanderhe) [reply sktools/src/sktools/scripts/collectwavecoeffs.py:161]: One could argue that this fact makes it follow the style-guide of the sktools, but you are right of course. Docstring added.

**Comment by bhourahine** [comment]
> > Seems fine, but I wasn't able to work out why a local copy of get_parser_and_subparser_container is required (other than possibly different comments/strings)..
> 
> It seemed to be the established strategy that at least some scripts provide their own definition. Doesn't mean it's a good design per se...

OK, but at some point we should discuss the style guide for Python and, as with the pylint stuff in #80, work out something that will be stuck to.

**Pull request merged**